Sources/CouchDB/CouchStructures/AllDatabaseDocuments.swift

```diff
@@ -16,8 +16,29 @@
 
 import Foundation
 
-/// A struct representing the JSON returned when querying a Database or View.
-/// http://docs.couchdb.org/en/stable/json-structure.html#all-database-documents
+/**
+ A struct representing the JSON returned when querying a Database or View.
+ If `includeDocuments` was true for the query, each row will have an additional "doc" field containing the JSON document.
+ These documents can then be decoded to a given swift type using `decodeDocuments(ofType:)`.
+ http://docs.couchdb.org/en/stable/json-structure.html#all-database-documents  
```

Comment:
It doesn't make sense having this link floating at the bottom of the paragraph.  You could either refer to it at the end of the first sentence, or have a few words around it at the bottom of the paragraph.